charts/aws-fluent-bit/values.yaml

```diff
@@ -26,7 +34,8 @@ readinessProbe:
     path: /api/v1/health
     port: http
 extraVolumes: []
-securityContext: {}
+securityContext:
+  runAsUser: 1000
```

Comment:
@patrick-stephens I had got the permission denied errors without setting this to the uid 1000. also when i exec'd to the pod and ran cat /etc/passwd, i didn't find any user accounts with uid as 1000. seems weird

Reply:
Usually this is down to the underlying host volumes - did you run on a host with UID 1000?

Reply:
yes.

Reply:
Usually then it is down to mounting configuration (filesystem security).
The container only has 9876 user I think: https://github.com/calyptia/calyptia-fluent-bit/blob/cbbf33b8311f1e00f8b30d2210ad533582528a6b/Dockerfile#L126

It is a Red Hat UBI base image and typically you need to mount in files with the appropriate context.